src/main/java/org/checkerframework/specimin/SpeciminRunner.java

```diff
@@ -425,13 +425,6 @@ private static void performMinimizationImpl(
     Set<String> updatedUsedClass = solveMethodOverridingVisitor.getUsedClass();
     updatedUsedClass.addAll(totalSetOfAddedInheritedClasses);
 
-    // remove the unsolved annotations in the newly added files.
-    for (CompilationUnit cu : parsedTargetFiles.values()) {
-      cu.accept(annoRemover, null);
```

Comment:
I believe my initial thought process was that it would remove all unsolved annotations before going into `AnnotationParameterTypesVisitor`, and then it would remove any `@Override`s in any files added by `AnnotationParameterTypesVisitor`. Now that I think of it, there shouldn't be any `@Override`s added so the last usage (which you commented on) should be its only usage to remove any possible unsolved annotations/`@Override`s. Thanks!